### PR TITLE
M1M3 Thermal System auxiliary devices

### PR DESCRIPTION
**Pull request by pkubanek**
* VFD Pump controller
* Flow meter

### CODE REVIEW

File: sal_interfaces/MTM1M3TS/MTM1M3TS_Events.xml [hunk 1]
```diff
@@ -515,4 +515,86 @@
       <Count>1</Count>
     </item>
   </SALEvent>
+  <SALEvent>
+    <Subsystem>MTM1M3</Subsystem>
+    <EFDB_Topic>MTM1M3_logevent_pumpStatus</EFDB_Topic>
```
Comment (rbovill) [inline sal_interfaces/MTM1M3TS/MTM1M3TS_Events.xml:520]: I believe this needs to be MTM1M3TS

**Review by rbovill — CHANGES_REQUESTED** [review]
At least one topic needs to be renamed slightly.

File: sal_interfaces/MTM1M3TS/MTM1M3TS_Events.xml (hunk 1 above)
Comment (pkubanek) [reply sal_interfaces/MTM1M3TS/MTM1M3TS_Events.xml]: You are correct. Fixed. Too many hunters - a rabbit dead, a Czech proverb.

**Review by rbovill — APPROVED** [review]
LGtM

**Review by rbovill — APPROVED** [review]
Changes look good, too.

**Pull request merged**